src/uk/gov/hmcts/contino/AzPrivateDns.groovy

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.contino
+
+import groovy.json.JsonOutput
+import uk.gov.hmcts.contino.azure.Az
+
+class AzPrivateDns {
+
+    def steps
+    def environment
+    def subscription
+    def az
+
+    // environment -> [resource_group, subscription]
+    private final Map<String, String[]> envToRgAndSubId = [
```

Comment:
you planning on cleaning this up? I feel like it could be a lot cleaner

Reply:
yes, missed that. thanks

Reply:
(discussing in slack)